apps/condo/domains/acquiring/schema/SumPaymentsService.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Generated by `createservice acquiring.SumPaymentsService`
+ */
+
+const { GQLCustomSchema } = require('@condo/keystone/schema')
+const access = require('@condo/domains/acquiring/access/SumPaymentsService')
+const { GqlWithKnexLoadList } = require('../../common/utils/serverSchema')
+const Big = require('big.js')
```

Comment:
please reorder imports,  

1. external imports (big.js)
2. internal imports from other modules (@condo/keystone)
3. local imports (@condo/domains or @condo/domains/common)

Also you need to use `@condo`, not `../..`